doc/update_banners.md

```diff
@@ -0,0 +1,58 @@
+# How to Update Banners
+
+This documentation provides instructions on how to update banners for the website using the `banners.json` file located at `src/assets/data/banners.json`.
+
+## File Structure
+
+The `banners.json` file has the following structure:
+
+```json
+{
+  "top": {
+    "enable": false,
+    "content": "This banner will be displayed on top of the website."
+  },
+  "event": {
+    "enable": true,
+    "title": "Green Code Initiative I/O 2025",
+    "content": "It's an awesome event!",
+    "action": {
+      "label": "I register",
+      "link": "https://green-code-initiative.org"
+    }
+  }
+}
+```
+
+## Updating Banners
+
+### Top Banner
```

Comment:
it could be great to have a screenshot in the documentation about how the top banner is displayed to avoid some tests just to remember how it's displayed ;)